src/main/java/seedu/bankwithus/Account.java

```diff
@@ -36,5 +47,8 @@ public void subtractBalance(float currentBalance, float withdrawal) {
         DecimalFormat df = new DecimalFormat("#.##");
         String formatted = df.format(currentBalance - withdrawal);
         this.balance = String.valueOf(formatted);
+
+        withdrawalChecker.updateTotalAmtWithdrawn(withdrawal);
```

Comment:
Called function to update withdrawals this month in the subtractBalance function